hcl/hcl_script_test.go

```diff
@@ -0,0 +1,659 @@
+						Mkrange("script.tm", Start(2, 29, 29), End(2, 30, 30))),
+					errors.E(hcl.ErrScriptMissingOrInvalidJob,
+						Mkrange("script.tm", Start(2, 29, 29), End(2, 30, 30))),
+				},
+			},
+		},
+		{
+			name: "script without a label",
+			input: []cfgfile{
+				{
+					filename: "cfg.tm",
+					body: `
+					terramate {
+						config {
+							experiments = ["scripts"]
+						}
+					}
+				`,
+				},
+				{
+					filename: "script.tm",
+					body: `
+					  script {
+						description = "some description"
+						job {
+						  command = ["echo", "hello"]
+						}
+					  }
+		`,
+				},
+			},
+			want: want{
+				errs: []error{
+					errors.E(hcl.ErrScriptNoLabels,
+						Mkrange("script.tm", Start(2, 8, 8), End(2, 14, 14))),
+				},
+			},
+		},
+		{
+			name: "script with a description attr",
+			input: []cfgfile{
+				{
+					filename: "cfg.tm",
+					body: `
+					terramate {
+						config {
+							experiments = ["scripts"]
+						}
+					}
+				`,
+				},
+				{
+					filename: "script.tm",
+					body: `
+					  script "group1" "script1" {
+						description = "some description"
+					  }
+					`,
+				},
+			},
+			want: want{
+				errs: []error{
+					errors.E(hcl.ErrScriptMissingOrInvalidJob,
+						Mkrange("script.tm", Start(2, 34, 34), End(2, 35, 35))),
+				},
+			},
+		},
+		{
+			name: "script with an unrecognized attr",
+			input: []cfgfile{
+				{
+					filename: "cfg.tm",
+					body: `
+					terramate {
+						config {
+							experiments = ["scripts"]
+						}
+					}
+				`,
+				},
+				{
+					filename: "script.tm",
+					body: `
+					  script "group1" "script1" {
+						description = "some description"
+						unknownattr = "abc"
+						job {
+						  command = ["ls"]
+						}
+					  }
+					`,
+				},
+			},
+			want: want{
+				errs: []error{
+					errors.E(hcl.ErrScriptUnrecognizedAttr,
+						Mkrange("script.tm", Start(4, 7, 81), End(4, 18, 92))),
+				},
+			},
+		},
+		{
+			name: "script with a description attr and job command",
+			input: []cfgfile{
+				{
+					filename: "cfg.tm",
+					body: `
+					terramate {
+						config {
+							experiments = ["scripts"]
+						}
+					}
+				`,
+				},
+				{
+					filename: "script.tm",
+					body: `
+					  script "group1" "script1" {
+						description = "some description"
+						job {
+						  command = ["echo", "hello"]
+						}
+					  }
+					`,
+				},
+			},
+			want: want{
+				config: hcl.Config{
+					Terramate: &hcl.Terramate{
+						Config: &hcl.RootConfig{
+							Experiments: []string{"scripts"},
+						},
+					},
+					Scripts: []*hcl.Script{
+						{
+							Labels:      []string{"group1", "script1"},
+							Description: hcl.NewScriptDescription(makeAttribute(t, "description", `"some description"`)),
+							Jobs: []*hcl.ScriptJob{
+								{
+									Command: makeCommand(t, `["echo", "hello"]`),
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "script with an unrecogznized child block of job",
```

Comment:
```suggestion
			name: "script with an unrecognized child block of job",
```

Reply:
Fixed in https://github.com/terramate-io/terramate/pull/1236/commits/0bd145bc88f507782997876f06b14dc3a983a24c